server/app/views/applicant/QuestionSummaryFragment.html

```diff
@@ -4,10 +4,9 @@
     <div class="border-t border-base-lighter content"></div>
 
     <div class="usa-prose container">
-      <!-- TODO(#8793): Add optional * indicator -->
```

Comment:
**question (non-blocking):** Since we haven't yet added the "(Optional)" indicator, maybe we should keep this TODO.  See #5510.

Reply:
Done